_pages/home.md

```diff
@@ -17,33 +17,34 @@ intro:
   - excerpt: "
 <div class='excerpt-title'>Latest News</div>
 
+<style> li{width:100%} </style>
 <ul>
   <li>
-    <a href='/release/pack-downloads'>Apache ServiceComb Pack 0.5.0 is released</a>
+      <a href='/release/'>Apache ServiceComb Release Version 1.2.0</a>
```

Comment:
'/release/' page is old, it should be updated

Reply:
thanks, I have fixed this problem in the new commit

Reply:
thanks, I have fixed this problem in the new commit, pls help to check